router/batchrouter/asyncdestinationmanager/lytics_bulk_upload/lyticsBulkUpload.go

```diff
@@ -0,0 +1,379 @@
+package lyticsBulkUpload
+
+import (
+	"bufio"
+	"encoding/csv"
+	"encoding/json"
+	stdjson "encoding/json"
```

Comment:
Can you use jsoniter instead of encoding/json

Reference: https://github.com/rudderlabs/rudder-server/blob/5327d4413e8aa92ab737243bb45c5971a39e5db8/router/batchrouter/asyncdestinationmanager/klaviyobulkupload/klaviyobulkupload.go#L31

Reply:
changed